docs/topics/mta-cli-args.adoc

```diff
@@ -0,0 +1,248 @@
+|--keepWorkDirs| Flag to instruct {ProductShortName} to not delete temporary working files, such as the graph database and extracted archive files. This is useful for debugging purposes.
+|--legacyReports| Flag to instruct {ProductShortName} to generate the old format reports instead of the new format reports.
+|--list| Flag to list installed add-ons.
+|--listSourceTechnologies| Flag to list all available source technologies.
+|--listTags| Flag to list all available tags.
+|--list-targets| Flag to list all available target technologies.
+|--mavenize| Flag to create a Maven project directory structure based on the structure and content of the application. This creates `pom.xml` files using the appropriate Java EE API and the correct dependencies between project modules. See also the `--mavenizeGroupId` option.
+|--mavenizeGroupId| When used with the `--mavenize` option, all generated `pom.xml` files will use the provided value for their `<groupId>`. If this argument is omitted, {ProductShortName} will attempt to determine an appropriate `<groupId>` based on the application, or will default to `com.mycompany.mavenized`.
+|--online |Flag to allow network access for features that require it. Currently only validating XML schemas against external resources relies on Internet access. Note that this comes with a performance penalty.
+|--output |Specify the path to the directory to output the report information generated by {ProductShortName}.
+|--overwrite |Flag to force delete the existing output directory specified by `--output`. If you do not specify this argument and the `--output` directory exists, you are prompted to choose whether to overwrite the contents.
+
+[IMPORTANT]
+====
+Do not overwrite a report output directory that contains important information.
+====
+
+|--packages| A space-delimited list of the packages to be evaluated by {ProductShortName}. It is highly recommended to use this argument.
+|--remove |Remove the specified add-ons. The syntax is `<GROUP_ID>:<ARTIFACT_ID>[:<VERSION>]`. For example, `--remove core-addon-x` or `--remove org.example.addon:example:1.0.0`.
+|--skipReports |Flag to indicate that HTML reports should not be generated. A common use of this argument is when exporting report data to a CSV file using `--exportCSV`.
+|--source |A space-delimited list of one or more source technologies, servers, platforms, or frameworks to migrate from. This argument, in conjunction with the `--target` argument, helps to determine which rulesets are used. Use the `--listSourceTechnologies` argument to list all available sources.
+|--sourceMode |Flag to indicate that the application to be evaluated contains source files rather than compiled binaries. The sourceMode argument has been deprecated. There is no longer the need to specify it. {ProductShortName} can intuitively process any inputs that are presented to it.  In addition, project source folders can be analyzed with binary inputs within the same analysis execution.
```

Comment:
This argument is now:

`-m, --mode string               analysis mode. Must be one of 'full' or 'source-only' (default "full")`

Reply:
Added line 58